bindings/python/examples/client/05_get_address_balance.py

```diff
@@ -35,8 +35,10 @@
 for output_with_metadata in outputs:
     output = output_with_metadata.output
     total_amount += output.amount
-    if output.native_tokens:
-        native_tokens.append(output.native_tokens)
+    native_token = [feature for feature in output.features if feature['type']
```

Comment:
Would `feature.type` work instead?

Reply:
Maybe, can't test at the moment since all is broken